src/test/java/org/folio/services/group/GroupExpenseClassTotalsServiceTest.java

```diff
@@ -264,21 +275,26 @@ void getExpenseClassTotalsMultipleRecords(VertxTestContext vertxTestContext) {
       .onComplete(result -> {
         var groupExpenseClassTotalsCollection = result.result();
         assertThat(groupExpenseClassTotalsCollection.getGroupExpenseClassTotals(), hasSize(2));
+
         GroupExpenseClassTotal expected1 = new GroupExpenseClassTotal()
           .withExpenseClassName(expenseClass1.getName())
           .withId(expenseClassId1)
           .withEncumbered(11.31)
           .withAwaitingPayment(3d)
-          .withExpended(95d)
-          .withPercentageExpended(9.5);
+          .withCredited(5d)
+          .withPercentageCredited(-100d)
```

Comment:
This should be 100d.

Reply:
![image](https://github.com/folio-org/mod-finance/assets/113523904/2b200e4a-17d1-4635-a9cf-c6455d0b7215)
I found that this method was returning minus value, this is removed. now tests updated